test/module/irohad/ordering/ordering_service_test.cpp

```diff
@@ -95,7 +109,8 @@ TEST_F(OrderingServiceTest, SimpleTest) {
 
   EXPECT_CALL(*fake_transport, publishProposal(_, _)).Times(1);
 
-  fake_transport->publishProposal(model::Proposal({}), {});
+  fake_transport->publishProposal(
+      shared_model::proto::TemplateProposalBuilder<7>().build(), {});
```

Comment:
Can this be replaced with TestTransactionBuilder?

Reply:
Validation can be applied, don't think so